models/business.model.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+var businessSchema = new mongoose.Schema({
+  // business_image_url: { type: String ,default: null},
+  image: { type: Buffer },
+  businessName: { type: String },
+  otherName: { type: String },
+  businessWebsite: { type: String },
```

Comment:
can u also mention required field in this? can we also have createdAt and lastUpdated fields?